blackmagic/db.py

```diff
@@ -45,6 +45,22 @@ def execute(cfg, stmt, keyspace=None):
                 conn['cluster'].shutdown()
 
                 
+def execute2(cfg, statement, parameters, keyspace=None):
```

Comment:
did you mean for 'execute2' to replace 'execute' ?

Reply:
execute2 is another arity for execution that accepts a string statement and parameters rather than a string statment+parameters.  The second arity is necessary to store binary data in Cassandra.  execute2 is not meant to supercide execute.  Open to better naming if you have suggestions.